content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Instrumentation
 description: >-
-  How OpenTelemetry facilitates automatic and manual instrumentation of
+  How OpenTelemetry facilitates the instrumentation of libraries and
```

Comment:
```suggestion
  How OpenTelemetry instrumentations libraries and
```

Reply:
hm, what you suggest lacks a verb, should it be How OpenTelemetry instruments libraries and applications.

Reply:
Oh, right, that was a misspelling on my part.